sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=CMCLFE
+sonar.projectName=CMC :: Legal Frontend
+sonar.projectVersion=0.0.1
+
+sonar.sources=src/main
+sonar.tests=src/test
+sonar.language=ts
+sonar.exclusions=src/test/**, src/main/resources/**, src/main/public/**/*,"src/main/views/*.njk", src/main/views/**, src/main/resources/**, src/main/modules/**
+sonar.dynamicAnalysis=reuseReports
+
+sonar.sourceEncoding=UTF-8
+sonar.ts.tslint.configPath=tslint.json
+sonar.ts.coverage.lcovReportPath=coverage/lcov.info
+sonar.host.url=https://sonar.reform.hmcts.net
```

Comment:
I don't think this should be set here

Reply:
I am not sure how we can set, just I followed footsteps of Probate and SSCS Frontend projects

Reply:
probably can do `sonar-scanner -Dsonar.host.url=${ENV_URL}`
Something like this:
https://github.com/hmcts/cmc-claim-store/blob/master/Jenkinsfile#L85

Reply:
Good spot @timja-kainos. It is possible by providing `SONAR_SCANNER_OPTS` e.g.

```
export SONAR_SCANNER_OPTS="-Dsonar.host.url=$SONARQUBE_URL"
```

Reply:
removed the host url and when we do ticket ROC-2326 we add this to jenkins stage